source/features/same-tab-links.tsx

```diff
@@ -0,0 +1,24 @@
+// ...existing code...
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function removeTargetBlank(anchor: HTMLAnchorElement): void {
+	anchor.removeAttribute('target');
+}
+
+function initOnce(): void {
+	observe('a[target="_blank"]', removeTargetBlank);
```

Comment:
As mentioned in my issue, I'd like to make this more specific, pointing to some specific links, not all links. You'll have to find the correct selector for that 
```suggestion
	observe([
     'the closed event a[target="_blank"]',
   ], removeTargetBlank);
```

Reply:
Hm, so I think we'd need a list of those specific ones or a modifiable list for that, yeah?

Reply:
Yes, but for now just one is fine, we can extend the list as needed.